udata_gouvfr/models.py

```diff
@@ -32,164 +32,193 @@
 Dataset.__badges__[NECMERGITUR] = _('Nec Mergitur')
 
 
-class ZonagesTownDataset(TerritoryDataset):
-    order = 8
-    id = 'zonages'
-    title = 'Zonages des politiques de la ville'
-    # Ministère de l'Egalité des Territoires et du Logement.
-    organization_id = '534fff8fa3a7292c64a77f3d'
-    url_template = (
-        'http://sig.ville.gouv.fr/Territoire/{code}/onglet/DonneesLocales')
+class PopulationDataset(TerritoryDataset):
+    order = 1
+    title = 'Population'
+    # Institut National de la Statistique et des Etudes Economiques (INSEE).
+    organization_id = '534fff81a3a7292c64a77e5c'
```

Comment:
We will need to find another pattern (later). Referencing and hard coding production identifier is not safe.